src/protocols/ec_udp.c

```diff
@@ -111,7 +111,7 @@ FUNC_DECODER(decode_udp)
           *
           * if the source is the ettercap host, don't display the message 
           */
-         if (ip_addr_is_ours(&PACKET->L3.src) == E_FOUND || ip_addr_is_ours(&PACKET->L3.src) == E_BRIDGE)
+         if (ip_addr_is_ours(&PACKET->L3.src) != E_FOUND && ip_addr_is_ours(&PACKET->L3.src) != E_BRIDGE)
             return NULL;
```

Comment:
Mhhh nope.
Given the purpose of the check described in the comment, the proposed change in this commit does the oposite.
Hence make no sense.